dev/io.openliberty.jakarta.messaging.3.0/bnd.bnd

```diff
@@ -8,9 +8,18 @@
 # Contributors:
 #     IBM Corporation - initial API and implementation
 #*******************************************************************************
--include= jar:${fileuri;${repo;jakarta.jms:jakarta.jms-api;3.0.0;EXACT}}!/META-INF/MANIFEST.MF,bnd.overrides
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.jakarta.messaging.3.0; singleton:=true
```

Comment:
```suggestion
Bundle-SymbolicName: io.openliberty.jakarta.messaging.3.0; singleton:=true
Bundle-Description: Jakarta Messaging 3.0 Specification
Bundle-Name: Jakarta Messaging API

Specification-Version: 3.0
```

The specification version is needed for knowing what version is being used for one of the JMS APIs.  An EE9 repeat fat likely fails without this update and the EE 9 CTS will fail without this change as well.  See #21464 for details.

Reply:
done